templates/base.html

```diff
@@ -23,10 +23,13 @@
                     <ul>
                         <li><a {% if page.title == "Programme" %} class="active"{% endif %}href="/programme/">Programme</a></li>
                         <li><a {% if page.title == "Venue" %} class="active"{% endif %}href="/venue/">Venue</a></li>
-                        <li><a {% if page.title == "Code Of Conduct" %} class="active"{% endif %}href="/code-of-conduct/">Conduct</a></li>
+                        <li><a {% if page.title == "Code Of Conduct" %} class="active"{% endif %}href="/code-of-conduct/">Code of Conduct</a></li>
                         <li class="halfway"><a {% if page.title == "Contact" %} class="active"{% endif %}href="/contact/">Contact</a></li>
                         <li><a {% if page.title == "Sponsorship" %} class="active"{% endif %}href="/sponsorship/">Sponsor</a></li>
                         <li><a {% if page.title == "Registration" %} class="active"{% endif %}href="/register/">Register</a></li>
+                        <li><a {% if page.title == "Call for proposals" %} class="active"{% endif %}href="/cfp/">Call for proposals</a></li>
+                        <li><a {% if page.title == "Diversity & accessibility" %} class="active"{% endif %}href="/diversity-accessibility/">Diversity & accessibility</a></li>
```

Comment:
If we've set the site's encoding to UTF-8 (meta tag in the `head`)then it should be unnecessary to encode this.